simplekv/transform/__init__.py

```diff
@@ -10,7 +10,8 @@
     store.put(key, value)
 
 The list of transformations will be applied from left to right
-to the 'local' data (i.e. the data passed to `put`). So in the example
+to the data passed to store, and the transformed data is forwarded
+to the decorated store. So in the example
```

Comment:
Sorry, I still can not parse this sentence. Maybe we can drop it?

Reply:
I think an explanation of the order in which the transformations are applied should remain.

What about
```
The list of transformations is applied from left to right to data passed :meth:`~ValueTransformingDecorator.put` or `~ValueTransformingDecorator.put_file`
and the transformed data is forwarded to the decorated store (`dstore` in the above example). Conversely, the inverse of these transformations is applied from right-to-left when after data has been read from the decorated store and before being returned to the caller via :meth:`~ValueTransformingDecorator.get`, `~ValueTransformingDecorator.get_file`, or `~ValueTransformingDecorator.open`.
```

If this is still not clear, please make an alternative proposal, which explains the semantics (including order) of transformations applied.

Reply:
How about

> The list of transformations is applied from left to right to the data passed to
:meth:`~ValueTransformingDecorator.put` or `~ValueTransformingDecorator.put_file`
and the transformed data is forwarded to the decorated store (`dstore` in the above
example). Conversely, the inverse of these transformations is applied from right-to-left 
when using :meth:`~ValueTransformingDecorator.get`, 
`~ValueTransformingDecorator.get_file`, or `~ValueTransformingDecorator.open` to read data from the decorated store.

?

Reply:
Sounds good. The new commit contains the text you proposed.